### PR TITLE
Rspec keywords syntax

### PR DESCRIPTION
**Pull request by Cosaquee**
Basic syntax for RSpec such as: 
- expect, allow, to, receive, and_return, with, eq, raise_error, expect, allow, to, to_not

### CODE REVIEW

**Comment by Cosaquee** [comment]
Please review

**Comment by ryansobol** [comment]
Thanks for the pull request! I'll review it right now. While I do that, please undo the changes made to package.json.

File: grammars/rspec.cson [hunk 1]
```diff
@@ -16,6 +16,14 @@
     'name': 'keyword.other.hook.rspec'
   }
   {
+    'match': '(?<!\\. )\\b(receive|and_return|with|eq|raise_error|be_nil|at_least|once|twice|exactly|times|at_most|exactly|and_throw|and_yield)\\b(?![?!])'
+    'name': 'keyword.other.hook.rspec'
+  }
+    {
```
Comment (ryansobol) [inline grammars/rspec.cson]: Could you fix the indentation please?

**Comment by ryansobol** [comment]
Also, can you please provide a sample rspec file that demonstrates these changes? Before and after screenshots would work too.

**Comment by Cosaquee** [comment]
Where should i put screenshots ?

**Comment by ryansobol** [comment]
screen shot 2016-01-08 at 9 18 55 am

**Comment by Cosaquee** [comment]
atom-dark-rspec-syntax-on

atom-darks-spec-syntax-off

**Comment by Cosaquee** [comment]
@paveldruzyak review please.

**Comment by paveldruzyak** [comment]
@Cosaquee sure, I will do it today later

**Comment by ryansobol** [comment]
@Cosaquee How goes incorporating my feedback about rolling back changes to `package.json` and fixing the indentation?

**Comment by Cosaquee** [comment]
@ryansobol Everythink is ready, imho.

**Comment by paveldruzyak** [comment]
@Cosaquee I will merge it soon

**Comment by Cosaquee** [comment]
@paveldruzyak what is the status of PR ? Will it be merged or not ?

**Pull request closed without merging**